toolchain/check/import.cpp

```diff
@@ -165,7 +165,8 @@ static auto CopySingleNameScopeFromImportIR(
     Map<SemIR::NameScopeId, SemIR::NameScopeId>* copied_namespaces,
     SemIR::ImportIRId ir_id, SemIR::InstId import_inst_id,
     SemIR::NameScopeId import_scope_id, SemIR::NameScopeId parent_scope_id,
-    SemIR::NameId name_id) -> std::pair<SemIR::NameScopeId, SemIR::InstId> {
+    SemIR::NameId name_id)
+    -> std::tuple<SemIR::NameScopeId, SemIR::InstId, bool> {
```

Comment:
It'd be useful to update the function comment to describe the new part of the return type. Also, maybe it's worth using a struct for this type? It looks like the same type is now returned from here and from `AddNamespace`.

Reply:
Updated comment and introduced a named return type (`NamespaceResult` - could be more verbose (`NamespaceInsertionResult`? `AddNamespaceResult` seemed ambiguous (could be a verb "add (namespace result)" or noun "(add namespace) result") so I shied away from that).

Reply:
Yeah, `NamespaceResult` seems fine given that it's scoped to this file.